src/ontology/mondo-edit.obo

```diff
@@ -537013,6 +537016,7 @@ xref: ICD10CM:P57.8 {source="Orphanet:529808"}
 xref: MEDGEN:1806573 {source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 xref: Orphanet:529808 {source="MONDO:equivalentTo"}
 xref: UMLS:C5575229 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1806573"}
+is_a: MONDO:0005527 {source="https://www.ncbi.nlm.nih.gov/books/NBK559120/"} ! toxic encephalopathy
```

Comment:
@katiermullen this source has a PMID (see end of the page), which is preferable. I will update